app/models/rubygem.rb

```diff
@@ -95,10 +95,9 @@ def self.by_downloads
   end
 
   def self.current_rubygems_release
-    if (g = find_by(name: "rubygems-update")) && (v = g.versions.release.indexed.latest.first)
-      v.number
-    else
-      "0.0.0"
+    if (g = find_by(name: "rubygems-update")) &&
```

Comment:
`g` is an awful variable name. also shouldn't need the parens around either of these statements. How about `rubygem`?